src/NServiceBus.Core.Tests/RedirectHelper.cs

```diff
@@ -12,6 +14,7 @@ public class RedirectHelper
     public void Initialize()
     {
         AppDomain.CurrentDomain.AssemblyResolve += CurrentDomain_AssemblyResolve;
+        Thread.CurrentThread.CurrentUICulture = CultureInfo.DefaultThreadCurrentUICulture = CultureInfo.InvariantCulture;
```

Comment:
do we really want to piggyback this on the RedirectHelper? What about creating a dedicated "Helper" class?